src/commandDetails/coin/transfer.ts

```diff
@@ -0,0 +1,122 @@
+import { container } from '@sapphire/framework';
+import { User } from 'discord.js';
+import {
+  CodeyCommandOptionType,
+  CodeyCommandDetails,
+  SapphireMessageExecuteType,
+  getUserFromMessage,
+  SapphireMessageResponseWithMetadata,
+} from '../../codeyCommand';
+import {
+  adjustCoinBalanceByUserId,
+  getCoinBalanceByUserId,
+  UserCoinEvent,
+} from '../../components/coin';
+import { getCoinEmoji } from '../../components/emojis';
+import { pluralize } from '../../utils/pluralize';
+
+const coinTransferExecuteCommand: SapphireMessageExecuteType = async (
+  client,
+  messageFromUser,
+  args,
+) => {
+  // First mandatory argument is receiving user
+  const receivingUser = <User>args['user'];
+  if (!receivingUser) {
+    throw new Error('please enter a valid user mention or ID for balance transfer.');
+  }
+
+  // Second mandatory argument is amount
+  const amount = <number>args['amount'];
+  if (typeof amount !== 'number') {
```

Comment:
We should also check that amount is an integer. We don't want someone transferring 69.4204204204 coins and making the leaderboard ugly with the decimals.